CONTRIBUTING.md

```diff
@@ -205,3 +205,12 @@ Run `git config user.email` to see your Git email, and verify it with [your GitH
 webpack is feature rich and documentation is a time sink. We
 greatly appreciate any time spent fixing typos or clarifying sections in the
 documentation.
+
+## Testing with Docker
+
+Make sure to have installed docker before you continue.
+
+```sh
+$ docker build -t webpack-cli .
```

Comment:
The installation kept on failing on my environment due to the protocol of the dependencies installed via git. 

Once I added the `+https` it worked:

`"yeoman-generator": "git+https://github.com/ev1stensberg/generator.git#Feature-getArgument"`

and

`"recast": "git+https://github.com/kalcifer/recast.git#bug/allowbreak",`